dashboard/controller/assignments/exportCSV.js

```diff
@@ -0,0 +1,92 @@
+// include libraries
+var superagent = require('superagent'),
+	async = require('async'),
+	common = require('../../helper/common');
+
+module.exports =  function exportCSV(req, res) {
+	var assignment = [];
```

Comment:
The variable name conflicts with the other variables in the same function, for example `assignment` on line 9.

Reply:
no that will work because of scope in javascript

Reply:
Yeah, it will work, but it's better to have a different name for clarity, for example `assignments` in this case.